ghu_web/ghu_main/templates/ghu_main/base.html

```diff
@@ -1,27 +1,22 @@
 {% extends 'ghu_global/layout.html' %}
-{% load static %}
 {% block site_name %}Global Humanitarians Unite{% endblock %}
 {% block styles %}
     {{ block.super }}
-    <link href="{% static "/ghu_main/css/base.css" %}" rel="stylesheet">
+    <style>
+        nav a {
+            /* Note this matches the number of nav links defined below */
+            width: 49%;
```

Comment:
so html is pretty annoying, so if you have two inline blocks like:

    <div id="a" />
    <div id="b" />

there will be a small whitespace node between the two, leaving an obnoxious space between them. this is why you had to write 49% above. if you do the following comment strategy which i wrote originally, you can set this width to 50%

       <div id="a" /><!--
    --><div id="b" />


i should have written a comment explaining why i was doing that, or said something when 8d9af20e removed them, apologies